magento/magento2
# Added attribute crossorigin to head.xsd for linkTypes

### Description (*)


Allows the crossorigin attribute to be added to `link` types, for HEAD elements, in as per https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin

### Related Pull Requests

N/A

### Fixed Issues (if relevant)


These issues/features are related:

1. https://github.com/magento/magento2/issues/32509
2. https://github.com/magento/magento2/issues/21427
3. https://github.com/magento/magento2/issues/29513

### Manual testing scenarios (*)

1. Add `<css src="https://my.cdn.com/style.min.css" media="screen" src_type="url" crossorigin="anonymous"/>` to default_head_blocks.xml
2. Ensure crossorigin attribute is added correctly and doesn't break the site with an XSD error

### Questions or comments


There's probably already some work in the pipeline regarding this work, but I thought I would create a PR anyway as I need the diff for a patch.

### Contribution checklist (*)
